SCClassLibrary/Common/Control/ServerStatus.sc

```diff
@@ -127,19 +126,13 @@ ServerStatusWatcher {
 			statusWatcher =
 			OSCFunc({ arg msg;
 				var cmd, one;
-				if(notify){
-					if(notified.not){
-						this.sendNotifyRequest;
-						"Receiving notification messages from server %\n".postf(server.name);
-					}
-				};
+				if(notify and: { notified.not }) { this.sendNotifyRequest };
 				alive = true;
 				#cmd, one, numUGens, numSynths, numGroups, numSynthDefs,
 						avgCPU, peakCPU, sampleRate, actualSampleRate = msg;
 				{
 					this.updateRunningState(true);
 					server.changed(\counts);
```

Comment:
I just added a simpler version of what I removed. The message is now part of `ServerStatusWatcher::notify_`, where it makes more sense.